mypy.ini

```diff
@@ -26,9 +26,6 @@ ignore_missing_imports = True
 [mypy-networkx.*]
 ignore_missing_imports = True
 
-[mypy-spdx.*]
-ignore_missing_imports = True
```

Comment:
don't we need to add `types-spdx` or something like that to the `setup.py`?

Reply:
they were added when I updated `spdx-tools` to 0.8.0, but forgot to remove it 😄